README.md

```diff
@@ -1,5 +1,15 @@
 # iscc-web - Minimal ISCC Generator Web Application
 
+# Configuration
+
+Configuration is handled by environment variables:
+
+**Backend:**
+
+- `ISCC_WEB_HOST`: defines on which host the HTTP server will bind to (default: localhost)
```

Comment:
The scheme, host, port env variable is not where the backend will bind to. It is the public site address and it is used to build absolute urls in api responses. See: 
https://github.com/iscc/iscc-web/blob/main/iscc_web/options.py#L24
https://github.com/iscc/iscc-web/blob/main/iscc_web/api/iscc.py#L46
https://github.com/iscc/iscc-web/blob/main/iscc_web/api/media.py#L25
https://github.com/iscc/iscc-web/blob/main/iscc_web/api/metadata.py#L83
...

Sorry for not adding documentation to  the environment variables.